ubyssey/templates/components/event-box.html

```diff
@@ -1,31 +1,31 @@
-<div class="c-event c-event--box{% if size %} c-event--{{ size}}{% endif %}">
-  {% if event.image and not hide_image %}
-  <a href="{% url 'event' event.pk.hex %}">
+<a href="{% url 'event' event.pk.hex %}">

```

Comment:
The only problem here is that `<a>` tags are `display: inline;` by default, so wrapping one around a block element might cause issues in certain browsers. I'd opt for something like this:

Convert the `div` to an `a` instead of wrapping it
```html
<a class="c-event c-event--box" href="{% url 'event' event.pk.hex %}">
  ...
</a>
```

```scss
.c-event--box {
  display: inline-block;
  ...
}